### PR TITLE
Fix kredits balance handling

### PR DESCRIPTION
**Pull request by bumi**
Kredits are stored on the Token contract as uint256 / bignumbers with 18
decimal points. Just like Ether and required by the ERC20 standard.
So we need to work with bignumbers and format a bignumber value here.

### CODE REVIEW

File: app/utils/cps/kredits.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { computed } from '@ember/object';
+import ethers from 'npm:ethers';
+//import formatKredits from 'kredits-web/utils/format-kredits.js';
```
Comment (bumi) [inline app/utils/cps/kredits.js]: how do I do that import here? somehow I did not get this working.

**Comment by skddc** [comment]
I completely disagree with this being added to kredits-web. We actually discussed this and agreed that it needs to be fixed in the wrapper, so that clients don't have to repeat all that implementation, and also because kredits are integers in the first place.

**Comment by bumi** [comment]
see that PR in kredits-contracts. 

on the ERC20 tokens we can not make sure that kredits are always full integers. 

in those lists here in kredits-web we must display the contribution kredits - not the token balances.

File: app/utils/cps/kredits.js (hunk 1 above)
Comment (fsmanuel) [reply app/utils/cps/kredits.js]: Without `.js` or a relative import: `import formatKredits from '../format-kredits';`

File: app/services/kredits.js [hunk 2]
```diff
@@ -109,7 +111,9 @@ export default Service.extend({
   },
 
   totalSupply: computed(function() {
-    return this.kredits.Token.functions.totalSupply();
+    return this.kredits.Token.functions.totalSupply().then(total => {
+      return Promise.resolve(formatKredits(total));
```
Comment (fsmanuel) [inline app/services/kredits.js]: No need for `Promise.resolve`: `return formatKredits(total);`

File: app/models/contributor.js [hunk 3]
```diff
@@ -1,12 +1,13 @@
 import { computed } from '@ember/object';
 import EmberObject from '@ember/object';
 import bignumber from 'kredits-web/utils/cps/bignumber';
```
Comment (fsmanuel) [inline app/models/contributor.js]: No longer needed, right?

**Comment by skddc** [comment]
> on the ERC20 tokens we can not make sure that kredits are always full integers.

How so? What good is an ERC20 config property for decimal points, if we cannot configure it the way we want?

File: app/models/contributor.js (hunk 3 above)
Comment (fsmanuel) [reply app/models/contributor.js]: Ah sorry, we still need it for the id?!

**Comment by bumi** [comment]
With this PR kredits-web with the current contracts would work. We could deploy that to have a working version? 
With the new PRs in kredits-contracts and an upcoming PR here we could switch to showing the earned kredits from contributions and not the token balances. (which imo would be more what we want in the UI right now). 

then we need to plan on how we show the different balances.

**Comment by skddc** [comment]
I don't think anyone objected to the merging of this PR.

**Comment by skddc** [comment]
Kredits label missing btw.

File: app/utils/cps/kredits.js (hunk 1 above)
Comment (bumi) [reply app/utils/cps/kredits.js]: ahhh, probably the only option that I did not try...thx.

**Comment by bumi** [comment]
does it create enough value for a kredits label :D

**Pull request merged**